patches/minecraft/net/minecraft/world/level/storage/loot/LootPool.java.patch

```diff
@@ -1,50 +1,24 @@
 --- a/net/minecraft/world/level/storage/loot/LootPool.java
 +++ b/net/minecraft/world/level/storage/loot/LootPool.java
-@@ -31,17 +_,19 @@
+@@ -31,6 +_,7 @@
  import org.apache.commons.lang3.mutable.MutableInt;
  
  public class LootPool {
--   final LootPoolEntryContainer[] f_79023_;
--   final LootItemCondition[] f_79024_;
 +   private final String name;
-+   private final List<LootPoolEntryContainer> f_79023_;
-+   private final List<LootItemCondition> f_79024_;
+    final LootPoolEntryContainer[] f_79023_;
+    final LootItemCondition[] f_79024_;
     private final Predicate<LootContext> f_79025_;
-    final LootItemFunction[] f_79026_;
-    private final BiFunction<ItemStack, LootContext, ItemStack> f_79027_;
+@@ -39,7 +_,8 @@
     NumberProvider f_79028_;
     NumberProvider f_79029_;
  
 -   LootPool(LootPoolEntryContainer[] p_165128_, LootItemCondition[] p_165129_, LootItemFunction[] p_165130_, NumberProvider p_165131_, NumberProvider p_165132_) {
--      this.f_79023_ = p_165128_;
--      this.f_79024_ = p_165129_;
 +   private LootPool(LootPoolEntryContainer[] p_165128_, LootItemCondition[] p_165129_, LootItemFunction[] p_165130_, NumberProvider p_165131_, NumberProvider p_165132_, String name) {
```

Comment:
Why is this private?

Reply:
That constructor signature change is part of the original patch and untouched by this PR, so a bouncer is unnecessary (as nothing calls this outside of this class as it is `private`). Furthermore, the change in modifier is because in 1.16.x, the unmodified constructor was `private`, but in 1.17.x, it is package-private, yet the patch was not changed during the update process to not add `private`.

Reply:
So it was a oversight during the update, and should be fixed properly in this pr...